plasma_framework/test/src/vaults/Erc20Vault.test.js

```diff
@@ -194,6 +194,42 @@ contract('Erc20Vault', (accounts) => {
         });
     });
 
+    describe('given quarantined exit game', () => {
```

Comment:
I would still put this under `withdraw`. Current test result/spec looks like this:
```
  withdraw
      ✓ should fail when not called by a registered exit game contract (40ms, 25079 gas)
      ✓ should transfer ERC token to the target (64ms, 33864 gas)
      ✓ should emit Erc20Withdrawn event correctly (45ms, 33864 gas)
  given quarantined exit game
      ✓ should fail when called under quarantine (41ms, 31294 gas)
      ✓ and then quarantine period passes (67ms, 33864 gas)
```
Which is a bit confusing on what's the target of the test without reading the implementation on the `given quarantined exit game` part

I think it would be easier to understand like this:
```
  withdraw
      ✓ should fail when not called by a registered exit game contract (40ms, 25079 gas)
      ✓ should transfer ERC token to the target (64ms, 33864 gas)
      ✓ should emit Erc20Withdrawn event correctly (45ms, 33864 gas)
      given quarantined exit game
          ✓ should fail when called under quarantine (41ms, 31294 gas)
          ✓ and then quarantine period passes (67ms, 33864 gas)
```

(PS. we can totally nest `describe`)

Reply:
Thx, it further simplifies test setup